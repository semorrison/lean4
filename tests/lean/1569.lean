def foo (x : Nat) (_ : x = 0) : Nat := x
example : foo 0 (by simp [typo]; done) = 0 := sorry
