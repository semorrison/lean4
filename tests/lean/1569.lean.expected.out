1569.lean:2:26-2:30: error: unknown identifier 'typo'
